charts/dvpe-deployment-gloo/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.1"
 description: Helm chart for installing microservices as gloo enabled VirtualService definitions.
 name: dvpe-deployment-gloo
-version: 1.2.2
+version: 1.2.3
```

Comment:
I would use at least 1.3.0 because the changes are more than a patch

Reply:
OK, will update to 1.3.0.